### ISSUE TITLE
Terminal buffer is visible underneath terminal accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#176971]
See the blue cursor

![Screenshot 2023-03-13 at 10 25 14 AM](https://user-images.githubusercontent.com/29464607/224747904-936c3739-1981-4045-8f1a-dd32ce7ef3cf.png)

**Comment by meganrogge** [comment]
This only reproduces when I use `Shift + Tab` - doesn't repro when I run the command `focus accessible buffer`

**Comment by meganrogge** [comment]
maybe related to this?

https://github.com/microsoft/vscode/blob/7800bd34921367f59448c0e4fcfedf2b58b44715/src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts#L85-L89

**Comment by meganrogge** [comment]
maybe the second condition should be `(e.keyCode === KeyCode.Tab && e.shiftKey)`

**Comment by meganrogge** [comment]
this was fixed
